File: lecture_001(Basic)/l_003_elseif.java
import java.util.Scanner;
public class l_003_elseif {
    public static Scanner scn= new Scanner(System.in);

    public static void main(String[] args) {
        System.out.print("Enter three values a,b and c:");
        int a= scn.nextInt();
        int b= scn.nextInt();
        int c= scn.nextInt();

        if (a>b && a>c) {
            System.out.print("a is greatest");
        }
        else if (b>a && b>c) {
            System.out.print("b is greatest");
        }
        else {
            System.out.print("c is greatest");
        }
    }
}

File: lecture_001(Basic)/l_003_elseif.cpp
#include<iostream>

using namespace std;

int main(int args, char**argvs) {
    cout<<"Enter three values a,b and c:"<<endl;
    int a,b,c;
    cin>>a>>b>>c;
    if (a>b && a>c) {
        cout<<"a is greatest";
    }

    else if(b>a && b>c) {
        cout<<"b is greatest";
    }
    else {
        cout<<"c is greatest";
    } 
    return 0;       
} 